scripts/dev/e2e.py

```diff
@@ -11,6 +11,9 @@
 import yaml
 
 TEST_POD_NAME = "e2e-test"
+TEST_CLUSTER_ROLE_NAME= "e2e-test"
+TEST_CLUSTER_ROLE_BINDING_NAME="e2e-test"
+TEST_SERVICE_ACCOUNT_NAME="e2e-test"
```

Comment:
We could just make this one constant, something like TEST_NAME. I left it like this for clarity but open to change it.